Packs/SplunkPy/ReleaseNotes/3_1_1.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### SplunkPy
+
+- Fixed an issue where the ***splunk-search*** command failed when the response contained a message from splunk-sdk.
```

Comment:
```suggestion
Fixed an issue where the ***splunk-search*** command failed when the response contained a message from splunk-sdk.
```

No need for a `-` prefix when there's only one item.

Reply:
Done